package/post-install.sh

```diff
@@ -26,28 +26,49 @@ ddtrace.request_init_hook=${EXTENSION_AUTO_INSTRUMENTATION_FILE}
 EOF
 }
 
-function install_conf_d_file() {
+function install_conf_d_files() {
     INI_FILE_PATH="${EXTENSION_CFG_DIR}/$INI_FILE_NAME"
 
-    println "Creating ddtrace.ini"
-    println "\n"
+    if [[ ! -f $INI_FILE_PATH ]]; then
+        println "Creating ddtrace.ini"
+        println "\n"
 
-    append_configuration_to_file "${INI_FILE_PATH}"
+        append_configuration_to_file "${INI_FILE_PATH}"
```

Comment:
seems like wrong tab/space creeped here

Reply:
It think the diff is weird looking, but that line is within an `if` block now. In my IDE it looks correct. Let me know if isn't show showing up correctly for you! :)